cpg-language-llvm/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/llvm/StatementHandler.kt

```diff
@@ -1391,9 +1391,8 @@ class StatementHandler(lang: LLVMIRLanguageFrontend) :
         // make an assignment in each BB.
         val functionName = LLVMGetValueName(bbsFunction).string
         val functions =
```

Comment:
can't we just use `tu.functions(functionName)` here?

Reply:
Yes. Changed